docs/application/native/api/iot-headed/5.0/group__Edje__Perspective.html

```diff
@@ -143,7 +143,7 @@
 <p>This function will return the perspective object that was set as global with <a class="el" href="group__Edje__Perspective.html#ga9e560272fb6b954a9e99eaad72a6755f" title="Makes this perspective object be global for its canvas.">edje_perspective_global_set()</a>.</p>
 <dl class="see"><dt><b>See also:</b></dt><dd><a class="el" href="group__Edje__Perspective.html#ga9e560272fb6b954a9e99eaad72a6755f" title="Makes this perspective object be global for its canvas.">edje_perspective_global_set()</a> </dd>
 <dd>
-<a class="el" href="group__Edje__Perspective.html#ga3dd9d197daf58ae5d4e4d37b2a05f6ee" title="Gets whether the given perspective object is global or not.">edje_perspective_global_get()</a> </dd></dl>
+<a class="el" href="group__Edje__Perspective.html#ga3dd9d197daf58ae5d4e4d37b2a05f6ee" title="Gets whether the given perspective object is global or not.">edje_perspective_global_get()</a>  2.3 </dd></dl>
```

Comment:
Would you please check the updates in this page?

![image](https://user-images.githubusercontent.com/9958374/102872559-35443880-4483-11eb-895e-1643c5b5f3dd.png)

![image](https://user-images.githubusercontent.com/9958374/102872582-3d9c7380-4483-11eb-9fec-567ca9522494.png)

![image](https://user-images.githubusercontent.com/9958374/102872599-45f4ae80-4483-11eb-8265-909227847209.png)

...and so on.

Reply:
I think.. it is bug generated by EFL versioning script.. Will check it with EFL team...

Reply:
https://review.tizen.org/gerrit/#/c/platform/upstream/efl/+/250479/